app/forms/BusinessWebsiteAddressForm.scala

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2022 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package forms
+
+import forms.FormValidation.{ErrorCode, nonEmptyValidText}
+import play.api.data.Form
+import play.api.data.Forms.{single, text}
+import uk.gov.hmrc.play.mappers.StopOnFirstFail
+
+import scala.util.matching.Regex
+
+object BusinessWebsiteAddressForm {
+
+  val businessWebsiteAddressKey = "businessWebsiteAddress"
+  implicit val errorCode: ErrorCode = businessWebsiteAddressKey
+  val regex: Regex = """^(((HTTP|http)(S|s)?\:\/\/((WWW|www)\.)?)|((WWW|www)\.))?[a-zA-Z0-9\[_~\:\/?#\]@!&'()*+,;=% ]+\.[a-zA-Z]{2,5}(\.[a-zA-Z]{2,5})?(\:[0-9]{1,5})?(\/[a-zA-Z0-9_-]+(\/)?)*$""".r
```

Comment:
This might not work with mixed cases for scheme part of url. Probably be better to use the case insensitive modifier of regex which should cover that case. thanks!